studio/src/pages/editor-page.tsx

```diff
@@ -414,17 +415,15 @@ export const EditorPage: React.FunctionComponent = () => {
                                     <Stack.Item>
                                         <OverflowSet
                                                 aria-label="Actions"
+                                                style={{ display: chatMode === 'TestMode' ? 'block': 'none' }}
                                                 overflowItems={[
                                                 {
-                                                    key: 'download',
-                                                    name: 'Download Transcripts',
-                                                    iconProps: { iconName: 'Download' },
-                                                    onClick: () => {},
-                                                }, {
-                                                    key: 'callbackForm',
-                                                    name: 'Callback form',
-                                                    iconProps: { iconName: 'FormLibrary' },
-                                                    onClick: () => {  },
+                                                    key: 'clearData',
```

Comment:
Don't remove Download Transcripts and Callback Forms

Reply:
![image](https://github.com/user-attachments/assets/cc9857cb-8a05-4cc2-86a4-52c415e5bbbc)

They are back.